pkgs/data/themes/tau-helium/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  meson,
+  dart-sass,
+  ninja
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tau-helium";
+  version = "1.5.5";
```

Comment:
Latest is 1.5.8 if you wanted to bump it now.

Reply:
@Lord-Valen ok, will bump the version

Reply:
Don't forget to change the commit message and PR title. :yum: